docs/implementation-guide.md

```diff
@@ -0,0 +1,161 @@
+
+### Additional References
+We have only provided a brief overview of PHDI, Building Blocks, and the pipelines we have designed. For additional information please refer to documents linked below.
+- [PHDI-google-cloud README](./README.md)
+- [PHDI-google-cloud Getting Started Guide](./getting_started.md)
+
+## Implementing the PHDI Google Cloud Platform Pipelines
+In this section we describe how a STLT can take this repository and use it to spin up all of the functionality that it offers in their own GCP environment.
+
+### Step 1: Prepare Your GCP Environment
+In order to proceed you will need either:
+- a GCP account with permissions to create new projects in your organization's GCP environment,
+or
+- a GCP account with `Owner` access to a project in your organizations’ GCP environment that was created to house the PHDI GCP pipelines, and the name of this project.
+
+If you do not meet either of these criteria contact the owner of your organization's GCP environment.
+
+### Step 2: Install the Gcloud CLI.
+The gcloud CLI is a command line tool provided by Google for working with GCP. We will use it to authenticate your local machine with your organization's GCP environment. Follow [this guide](https://cloud.google.com/sdk/docs/install) to install gcloud. After installation run `gcloud auth application-default login` and follow the prompts in your browser to login to GCP.
+
+### Step 3: Fork the phdi-google-cloud Repository
+Fork the phdi-google-cloud repository into your organization's, or your personal, GitHub account.
+1. Navigate to [https://github.com/CDCgov/phdi-google-cloud](https://github.com/CDCgov/phdi-google-cloud).
+2. Click on the `Fork` button in the top right.
+![fork-repo-1](./images/fork-repo-1.png)
+3. Select the owner of the forked repository. We recommend that this is an organization and not an individual.
+4. Click `Create fork` at the bottom of the page.
+![fork-repo-2](./images/fork-repo-2.png)
+
+### Step 4: Clone the Forked Repository
+Clone the forked version of the phdi-google-cloud repository by running `git clone https://github.com/<MY-GITHUB-ORGANIZATION>/phdi-google-cloud.git`. If you do not have `git` installed please follow [this guide](https://github.com/git-guides/install-git) to install it.
+
+### Step 5: Run the Quick Start Script
+In this step we will work through GCP's [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation) to granted your phdi-google-cloud repo access to deploy the pipelines to your organization's GCP environment. We have provided a script to automate most of this process that we recommend you use. However if you prefer to work through it manually you may follow [this guide](https://github.com/google-github-actions/auth#setup).
+
+From your machine's command line:
+1. Navigate to the root directory of the repository you cloned in step 4.
+2. Login into GCP by running `gcloud auth application-default login`.
```

Comment:
This is already mentioned in step 2 - and also, the quick start script actually runs `gcloud auth login`. Should we keep it as part of the script, or as step in the guide?

Reply:
Let's keep it as part of the script. We are suggesting that users use the script and this would automate away one additional task.

Reply:
Should we change the line in the script to run `gcloud auth application-default login` instead of `gcloud auth login`?

Reply:
Removed all `gcloud auth` steps from the IG. I'll look back at the script to see if we should make that change.